index.html

```diff
@@ -39,17 +39,143 @@ <h3 class="connect">LET'S CONNECT</h3>
+          <button class="see-project">See Project</Button>
+        </div>
+      </div>
+
+      <div class="projects">
+        <img src="./images/Snapshoot Portfolio(3).png" alt="">
+        <h2 class="works-title">TONIC</h2>
+        <p><span class="canopy">Canopy</span> <img src="./images/Counter.png" alt="Counter"> <span class="dev"> End Dev
+            <img src="./images/Counter.png" alt="Counter"> 2015</span></p>
+        <div class="project-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div>
+          <button class="languages">HTML</button>
+          <button class="languages">CSS</button>
+          <button class="languages">JAVASCRIPT</button>
+        </div>
+        <div>
+          <button class="see-project">See Project</Button>
+        </div>
+      </div>
+
+      <section class="about">
+        <div>
+          <div class="first-heading">
+            <h2>About Myself</h2>
+          </div>
+          <div class="description">
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+              work
+              and
+              experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+          </div>
+          <div>
+            <h3 class="connect">LET'S CONNECT</h3>
+            <ul class="social">
+              <li class="social"><a href="#"><img src="./images/Vector.svg" alt=""></a></li>
+              <li class="social"><a href="#"><img src="./images/Linkedin icon.svg" alt="Linkedin"></a></li>
+              <li class="social"><a href="#"><img src="./images/Vector (1).svg" alt=""></a></li>
+              <li class="social"><a href="#"><img src="./images/Vector (2).svg" alt="github"></a></li>
+              <li class="social"><a href="#"><img src="./images/Vector (3).svg" alt="twitter"></a></li>
+            </ul>
+          </div>
+          <div>
+            <button class="see-project">Get my resume</Button>
+          </div>
+          <ul class="union-list">
+            <li class="union-div">
+              <h2 class="union-header">Languages</h2>
+              <img class="union1" src="./images/Union.png" alt="">
+            <li class="about-lang">
+              <img class="union" src="./images/Ellipse 1.png" alt="">
+              <a href="http://#">JavaScript</a>
+            </li>
+            <li class="about-lang">
+              <img class="union" src="./images/Ellipse 1 (1).png" alt="">
+              <a href="http://#">HTML</a>
+            </li>
+            <li class="about-lang">
+              <img class="union" src="./images/Ellipse 1 (2).png" alt="">
+              <a href="http://#">CSS</a>
+            </li>
+          </ul>
+          </li>
+          <li class="union-div">
+            <h2>Frameworks</h2>
+            <img class="union2" src="./images/Union2.png" alt="">

```

Comment:
Your design
![image](https://github.com/MargaretTechworld/Portfolio/assets/78251927/cc20507a-28ee-4b50-b911-44081b437278)

Figma
![image](https://github.com/MargaretTechworld/Portfolio/assets/78251927/5a0d2b56-43cd-49be-a29c-2b3739f79526)

- As suggested by the previous reviewer, It would be nice if we reduce the font size and sizes of the elements in this section to match the template on Figma. 👍

- According to the project requirement, it is important that we stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- Also part of our learning objective while building this website is to understand how to parse a Figma design to create a UI. 🛠🤗